Source/WebCore/platform/audio/AudioSession.cpp

```diff
@@ -168,7 +168,7 @@ void AudioSession::setCategoryOverride(CategoryType category)
 
     m_categoryOverride = category;
     if (category != CategoryType::None)
-        setCategory(category, RouteSharingPolicy::Default);
+        setCategory(category, mode(), RouteSharingPolicy::Default);
 }
```

Comment:
This seems dangerous. Maybe we should instead just pass in Default in this case?